queue_services/entity-emailer/src/entity_emailer/email_processors/continuation_in_notification.py

```diff
@@ -118,28 +136,29 @@ def _get_pdfs(
                 }
             )
             attach_order += 1
+
     return pdfs
 
 
 def process(email_info: dict, token: str) -> dict:  # pylint: disable=too-many-locals, , too-many-branches
     """Build the email for Continuation notification."""
     logger.debug('filing_notification: %s', email_info)
-    # get template and fill in parts
+
+    # get template vars from email info
     filing_type, status = email_info['type'], email_info['option']
+
     # get template vars from filing
     filing, business, leg_tmz_filing_date, leg_tmz_effective_date = get_filing_info(email_info['filingId'])
     filing_name = filing.filing_type[0].upper() + ' '.join(re.findall('[a-zA-Z][^A-Z]*', filing.filing_type[1:]))
     filing_data = (filing.json)['filing'][f'{filing_type}']
     if status == Filing.Status.PAID.value:
         business = filing_data['nameRequest']
         business['identifier'] = filing.temp_reg
-
-    template = Path(f'{current_app.config.get("TEMPLATE_PATH")}/CONT-IN-{status}.html').read_text()
-    filled_template = substitute_template_parts(template)
-    # render template with vars
     legal_type = business.get('legalType')
     numbered_description = Business.BUSINESSES.get(legal_type, {}).get('numberedDescription')
-    jnja_template = Template(filled_template, autoescape=True)
+    review_result = ReviewResult.get_last_review_result(filing.id)
+    # encode newlines in review comment only
+    latest_review_comment = review_result.comments.replace('\n', '\\n') if review_result else None
```

Comment:
I did this to "encode" the line break, because the Jinja render engine otherwise removes them.

On line 192 I "decode" the line breaks.

Sample of a multi-line staff review comment:

![image](https://github.com/user-attachments/assets/b722235d-af12-481d-9e9a-70ee5cd9478b)